scm/driver/harness/pr.go

```diff
@@ -234,9 +229,21 @@ func convertPullRequest(src *pr) *scm.PullRequest {
 			ID:    src.Author.UID,
 			Email: src.Author.Email,
 		},
-		Fork:   "fork",
-		Ref:    fmt.Sprintf("refs/pullreq/%d/head", src.Number),
-		Closed: src.State == "closed",
+		Head: scm.Reference{
+			Name: src.SourceBranch,
+			Path: scm.ExpandRef(src.SourceBranch, "refs/heads"),
+			Sha:  src.SourceSHA,
+		},
+		Base: scm.Reference{
+			Name: src.TargetBranch,
+			Path: scm.ExpandRef(src.TargetBranch, "refs/heads"),
+			Sha:  src.MergeSHA,
```

Comment:
should it be merge sha or mergebasesha?

Reply:
I assume that's the sha of the target branch? if that's the case then it should be `MergeTargetSHA` - that's the SHA of the target branch when the last merge check was performed (if no merge check was performed - either it didn't happen yet or target was updated and it was reset - this value is nil btw)

Reply:
what is MergeBaseSha - i think it should be `mergebasesha` in that case - i saw github they are using base.sha but i dont know what the value points to

Reply:
probably the head sha when PR was raised?

Reply:
>probably the head sha when PR was raised?

I don't think we can use that, it'll  be outdated quite soon.
We should use what I mentioned.

`mergebasesha` is the sha that was the biggest common ancestor commit between target and source branch that was used at time of mergecheck.
We shouldn't use that either - could be far in the past if it's an older PR that wasn't rebased.

As I said, if we want the latest sha of target branch that was used for mergecheck then it's MergeTargetSHA.
If we want latest latest target sha - that we don't have here.